backend/tests/integration/services/adapters/message_bus/test_rabbitmq.py

```diff
@@ -338,41 +335,6 @@ async def test_rabbitmq_initial_setup(rabbitmq_api: RabbitMQManager) -> None:
     )
 
 
-async def test_rabbitmq_publish(rabbitmq_api: RabbitMQManager) -> None:
-    """Validate that the adapter publishes messages to the correct queue"""
-
-    bus = RabbitMQMessageBus(settings=rabbitmq_api.settings)
-    service = InfrahubServices(message_bus=bus, component_type=ComponentType.API_SERVER)
-
-    normal_message = messages.EventBranchCreate(branch="normal", branch_id=str(uuid4()), sync_with_git=False)
-    delayed_message = messages.EventBranchCreate(branch="delayed", branch_id=str(uuid4()), sync_with_git=False)
-
-    await bus.initialize(service=service)
-    await service.send(message=normal_message)
-    await service.send(message=delayed_message, delay=MessageTTL.FIVE)
-
-    queue = await bus.channel.get_queue(name=f"{bus.settings.namespace}.rpcs")
-    delayed_queue = await bus.channel.get_queue(name=f"{bus.settings.namespace}.delay.five_seconds")
-    message_from_queue = await queue.get()
-    delayed_message_from_queue = await delayed_queue.get()
-    parsed_message = ujson.loads(message_from_queue.body)
-    parsed_delayed_message = ujson.loads(delayed_message_from_queue.body)
-
-    await bus.shutdown()
-
-    parsed_message = messages.EventBranchCreate(**parsed_message)
-    parsed_delayed_message = messages.EventBranchCreate(**parsed_delayed_message)
-
-    # The priority isn't currently included in the header, reset it to show expected priority
-    normal_message.meta.priority = 3
-    delayed_message.meta.priority = 3
-    parsed_delayed_message.meta.headers = {"delay": 5000}
-    assert message_from_queue.priority == 5
-    assert delayed_message_from_queue.priority == 5
-    assert parsed_message == normal_message
-    assert parsed_delayed_message == delayed_message
```

Comment:
We'd still want this test to remain. While it's using the message for EventBranchCreate, what the test is actually validating is that our integration with RabbitMQ is working and that we can publish messages to RabbitMQ and then later read them from the queue. Instead of deleting this we could replace the EventBranchCreate message with something else such as SendEchoRequest or RefreshRegistryBranches.

Reply:
Definitely, actually we might want to reuse `messages.EventBranchCreate` when we change behavior of the queue in message bus so I am restoring the test as is (and using `SendEchoRequest`  / `RefreshRegistryBranches` would not test `PRIORITY_MAP` entry).